idaes/models/properties/activity_coeff_models/activity_coeff_prop_pack.py

```diff
@@ -209,6 +210,31 @@ def build(self):
                     initialize=1.0,
                     doc="Binary interaction parameter for " "Wilson model",
                 )
+        self.set_default_scaling("flow_mol", 1e-3)
+        self.set_default_scaling("mole_frac_comp", 10)
+        self.set_default_scaling("temperature", 1e-1)
+        self.set_default_scaling("temperature_dew", 1e-1)
+        self.set_default_scaling("temperature_bubble", 1e-1)
+        self.set_default_scaling("flow_mol_phase", 1e-2)
+        self.set_default_scaling("density_mol", 1)
+        self.set_default_scaling("pressure", 1e-6)
+        self.set_default_scaling("pressure_sat", 1e-6)
+        self.set_default_scaling("pressure_dew", 1e-6)
+        self.set_default_scaling("pressure_bubble", 1e-6)
+        self.set_default_scaling("mole_frac_phase_comp", 10)
+        self.set_default_scaling("enth_mol_phase_comp", 1e-3, index="Liq")
+        self.set_default_scaling("enth_mol_phase_comp", 1e-4, index="Vap")
+        self.set_default_scaling("enth_mol_phase", 1e-3, index="Liq")
+        self.set_default_scaling("enth_mol_phase", 1e-4, index="Vap")
+        self.set_default_scaling("entr_mol_phase_comp", 1e-2)
+        self.set_default_scaling("entr_mol_phase", 1e-2)
+        self.set_default_scaling("mw", 100)
+        self.set_default_scaling("mw_phase", 100)
+        self.set_default_scaling("gibbs_mol_phase_comp", 1e-3)
+        # Check values for these 3
+        self.set_default_scaling("fug_vap", 1)
```

Comment:
This needs the same scaling as pressure (more or less). It is `pressure*fugacity_coefficient`.